### ISSUE TITLE
perfdata should be single line

### ISSUE THREAD

**Issue by mpounsett** [mpounsett/nagiosplugin#11]
**[Original report](https://bitbucket.org/flyingcircus/nagiosplugin/issue/11) by Onur Yalazı (Bitbucket: [yalazi](https://bitbucket.org/yalazi), ).**

----------------------------------------

Nagios and icinga2 uses only a single line output to get check results and performance data. In case of multiple metrics *output.py:format_perfdata* method joins metrics with newlines. This breaks perf data collection. This should join with spaces.

Sample output:
```
ONLINE OK - Online is 2528
| 'www.xxxx.com'=74;;;0 'www.yyyy.com'=3;;;0 'www.zzzz.com'=306;;;0
'www.ttttt.com'=80;;;0 'www.bbbbbb.com'=819;;;0
```

**Comment by mpounsett** [comment]
**Original comment by Christian Kauhaus (Bitbucket: [ckauhaus](https://bitbucket.org/ckauhaus), GitHub: [ckauhaus](https://github.com/ckauhaus)).**

----------------------------------------

Calling a plugin with verbose level 0 will always return all results in one line. I think it would be resonable to change output behaviour for verbose levels >0 to still keep perfdata in one line.

**Comment by mpounsett** [comment]
**Original comment by Christian Kauhaus (Bitbucket: [ckauhaus](https://bitbucket.org/ckauhaus), GitHub: [ckauhaus](https://github.com/ckauhaus)).**

----------------------------------------

Always keep perfdata on a single line

Some systems swallow multi-line performance data even if output style is
perfectly ok w.r.t. the Nagios plugin specs.

So we restrict ourselves to single line performance data even in multi-line
mode.

Fixes #11
